web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -63,17 +65,24 @@ export class GridComponentRenderer {
     // The value of field will be precipForecast, rhForecast, tempForecast, etc.
     // We need the prefix to help us grab the correct 'actual' field (eg. tempACTUAL, precipACTUAL, etc.)
     const actualField = this.getActualField(field)
+    const isActual = !isNaN(params.row[actualField])
 
     const isGrassField = field.includes('grass')
-
-    const isActual = !isNaN(params.row[actualField])
+    // We can disable a cell if an Actual exists or the forDate is before today.
+    // Both forDate and today are currently in the system's time zone
```

Comment:
I know this is relatively simple, but could we pull this out into a helper function? Would make it easier to test and keep `renderForecastCellWith` focused on the rendering.

Reply:
Yep I can do that. I was also thinking the logic on line 85, setting `value` might be better to put in the `predictionItemValueFormatter`....it would keep formatting in the Formatter. Worth changing?

Reply:
Yeah I think that's a good call too